lib/stack_master/config.rb

```diff
@@ -3,11 +3,15 @@
 
 module StackMaster
   class Config
+    ConfigParseError = Class.new(StandardError)
+
     def self.load!(config_file = 'stack_master.yml')
       resolved_config_file = search_up_and_chdir(config_file)
       config = YAML.load(File.read(resolved_config_file))
       base_dir = File.dirname(File.expand_path(resolved_config_file))
       new(config, base_dir)
+    rescue Psych::SyntaxError => error
+      raise ConfigParseError, "Unable to parse stack_master.yml: #{error}"
```

Comment:
stack_master.yml is hardcoded in the message. Should it be `#{config_file}` instead?

Reply:
Yes, it should `stack_master.yml` anyways, but using the `resolved_config_file` would probably help more too. See d62f9ba